spec/api/keys/keys_spec.rb~

```diff
@@ -0,0 +1,14 @@
+# Copyright: Copyright (c) 2014 Chef, Inc.
```

Comment:
Fuuuuuuuuu I forgot to re-setup global git ignore post new hard drive. All those little things...